EventHubSampleData/EventHubSampleSendData/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Azure.Messaging.EventHubs;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Producer;
+
+namespace SendSampleData
+{
+    class Program
+    {
+        const string eventHubName = "test-hub";
+        // Copy the connection string ("Connection string-primary key") from your Event Hub namespace.
+        const string connectionString = @"<YourConnectionString>";
+
+        public static async Task Main(string[] args)
+        {
+            await EventHubIngestionAsync();
+        }
+
+        public static async Task EventHubIngestionAsync()
+        {
+            await using (var producerClient = new EventHubProducerClient(connectionString, eventHubName))
+            {
+                int counter = 0;
+                for (int i = 0; i < 100; i++)
+                {
+                    int recordsPerMessage = 3;
+                    try
+                    {
+                        List<string> records = Enumerable
+                            .Range(0, recordsPerMessage)
+                            .Select(recordNumber => $"{{\"timeStamp\": \"{DateTime.UtcNow.AddSeconds(100 * counter)}\", \"name\": \"{$"name {counter}"}\", \"metric\": {counter + recordNumber}, \"source\": \"EventHubMessage\"}}")
+                            .ToList();
```

Comment:
nit: `ToList` isn't needed here since `string.Join` accepts an enumerable.  Doesn't hurt anything, but we may want to remove it for succinctness.